File: src/main/java/com/polygloat/dtos/request/ResetPassword.java
package com.polygloat.dtos.request;

import lombok.Data;

import javax.validation.constraints.Max;
import javax.validation.constraints.Min;
import javax.validation.constraints.NotBlank;

@Data
public class ResetPassword {
    @NotBlank
    String email;

    @NotBlank
    String code;

    @Min(8)
    @Max(100)
    String password;
}

File: src/main/java/io/polygloat/dtos/request/ResetPassword.kt
package io.polygloat.dtos.request

import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotBlank

data class ResetPassword(
        @NotBlank
        var email: String? = null,

        @NotBlank
        var code: String? = null,

        @Min(8) @Max(100)
        var password: String? = null,
)